test/integration/sql.test.ts

```diff
@@ -220,4 +220,26 @@ describe('SQL proxy', () => {
     expect(team).toBeDefined();
     expect(team?.name).toBe('Team ships 2');
   });
+
+  test("calling xata.sql as a function throws an error because it's not safe", async () => {
+    // @ts-expect-error - Testing invalid usage
+    await expect(xata.sql('SELECT * FROM teams')).rejects.toThrow(
+      'Invalid usage of `xata.sql`. Please use it as a tagged template or with an object.'
+    );
+  });
+
+  test('calling xata.sql with invalid prepared statement', async () => {
+    const order = 'ASC';
+    await expect(xata.sql<TeamsRecord>`SELECT * FROM teams ORDER BY name ${order}`).rejects.toThrow(
+      'invalid SQL: unused parameters: used 0 of 1 parameters'
+    );
+  });
+
+  test("calling xata.sql with invalid prepared statement doesn't throw an error when bypassing prepared statement protection", async () => {
+    const order = 'ASC';
+    const { records } = await xata.sql<TeamsRecord>({
+      statement: `SELECT * FROM teams ORDER BY name ${order}`
+    });
```

Comment:
Could you also add a test showing sending prepared parameters?

Reply:
We already had one test for that case in the file:

```
    const { records, warning, columns } = await xata.sql<TeamsRecord>({
      statement: `INSERT INTO teams (name) VALUES ($1) RETURNING *`,
      params: ['Team ships 2']
    });
```